### PR TITLE
Prepare sample / test data

### PR DESCRIPTION
**Pull request by WasilewM**

### CODE REVIEW

File: databases/dummy_users_data.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+use devices;
+
+INSERT INTO Users (comments) VALUES ('someone registered with Google OAuth');
```
Comment (MA3CIN) [inline databases/dummy_users_data.sql:3]: This is probably handled already by the identity provider (Azure B2C in our case, probably)

**Review by MA3CIN — COMMENTED** [review]
We need to converge the Data model ASAP, but i don't have time until Friday...
Looks good to me (other than the predictors as a separate device..?), but this is subject to change if we change the data model

File: databases/devices_db_schema.sql [hunk 2]
```diff
@@ -17,6 +17,7 @@ CREATE TABLE Models (
 	id INT AUTO_INCREMENT PRIMARY KEY
 	, manufacturer INT NOT NULL
 	, category INT NOT NULL
+    , energy_consumption FLOAT NOT NULL
 	, name VARCHAR(200) NOT NULL
 	, comments VARCHAR(1000)
 );
```
Comment (ASypula) [inline databases/devices_db_schema.sql:23]: (cannot post a comment lower in the code)
What do you think about storing the units of the data (jednostki)? e.g. celsius, wats, etc?

Comment (WasilewM) [reply databases/devices_db_schema.sql:23]: Well, when more an more parameters are to be introduced then we should change the schema to allow for that (generic database schema).  Do we want to do that? Or do we want to add just a few params to choose from?

Comment (ASypula) [reply databases/devices_db_schema.sql:23]: Add only a few params to choose from ;) I'd rather not make it too complicated

File: databases/statistics_microservice_ERD.png
Comment (ASypula) [inline databases/statistics_microservice_ERD.png:1]: In this ERD, the models and registered_devices do not need a relation between them presented?

File: databases/dummy_users_data.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+INSERT INTO Users (comments) VALUES ('someone registered with Google OAuth');
+INSERT INTO Users (comments) VALUES ('someone registered with Google OAuth');
+INSERT INTO Users (comments) VALUES ('someone registered with Google OAuth');
+COMMIT;
+
+INSERT INTO Registered_Devices (model, owner, x_position, y_position) VALUES (1, 1, 10, 10);
```
Comment (ASypula) [inline databases/dummy_users_data.sql]: Should those inserts be modified after removing the x and y position from registered_devices?

File: databases/geolocation_microservice_db_schema.sql [hunk 4]
```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE IF NOT EXISTS geolocation_microservice;
```
Comment (ASypula) [inline databases/geolocation_microservice_db_schema.sql]: As we have a database 'devices", can we either use the geolocation name here? Or maybe change the name of the devices database?

File: databases/geolocation_microservice_db_schema.sql [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+use geolocation_microservice;
+
+CREATE TABLE Users(
+	id INT AUTO_INCREMENT PRIMARY KEY
+	, comments VARCHAR(1000)
+);
+
+# this table will be cached in statistics microservice
+CREATE TABLE Registered_Devices (
```
Comment (ASypula) [inline databases/geolocation_microservice_db_schema.sql]: Shouldn't this table be present only in device database?

File: databases/geolocation_microservice_db_schema.sql (hunk 4 above)
Comment (WasilewM) [reply databases/geolocation_microservice_db_schema.sql]: Good idea - I've simplified DBs names not to have "microservice" suffix

File: databases/statistics_microservice_ERD.png
Comment (WasilewM) [reply databases/statistics_microservice_ERD.png:1]: We should, thanks for finding that

File: databases/dummy_users_data.sql (hunk 3 above)
Comment (WasilewM) [reply databases/dummy_users_data.sql]: Ofc that they should be removed

**Pull request merged**